main.nf

```diff
@@ -238,8 +238,8 @@ process setParalogAnalysis {
     echo "NF internal work directory for [setParalogAnalysis (${task.index})] process:\n\$processDir\n" > $logs_folder/wgd_paralogs.log
 
     if [ \${paranome} = "no" ] && [ \${colinearity} = "no" ]; then
```

Comment:
Check capitalization-independent? (But see below.)

Reply:
`/ERROR/` pattern is case-sensitive: if I replace it here with `/error/` the code ignores the error lines containing `ERROR` and the error box will not show them.

Reply:
This was referring to the "no"'s in `if [ \${paranome} = "no" ] && [ \${colinearity} = "no" ]` not the "ERROR", but I guess this isn't relevant anymore now, since the code moved into the Python scripts.

Reply:
Ouch, sorry. Indeed.